<ui version="4.0" >
 <class>TMSPreferencesDialog</class>
 <widget class="QDialog" name="TMSPreferencesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TMS servers setup</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="grpWmsServers" >
     <property name="title" >
      <string>TMS Servers</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>1</number>
      </property>
      <property name="leftMargin" >
       <number>1</number>
      </property>
      <property name="topMargin" >
       <number>1</number>
      </property>
      <property name="rightMargin" >
       <number>1</number>
      </property>
      <property name="bottomMargin" >
       <number>1</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2" >
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>5</number>
         </property>
         <property name="leftMargin" >
          <number>5</number>
         </property>
         <property name="topMargin" >
          <number>5</number>
         </property>
         <property name="rightMargin" >
          <number>5</number>
         </property>
         <property name="bottomMargin" >
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="label_10" >
           <property name="text" >
            <string>Server list:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="lvTmsServers" />
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame" >
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QLabel" name="label_11" >
           <property name="text" >
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edTmsName" >
           <property name="minimumSize" >
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Server adress:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edTmsAdr" />
         </item>
         <item>
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edTmsPath" />
         </item>
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Tile size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbTileSize" >
           <property name="maximum" >
            <number>9999</number>
           </property>
           <property name="singleStep" >
            <number>64</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Minimum zoom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbMinZoom" />
         </item>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Maximum zoom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbMaxZoom" />
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btApplyTmsServer" >
           <property name="text" >
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btAddTmsServer" >
           <property name="text" >
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btDelTmsServer" >
           <property name="text" >
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TMSPreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TMSPreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
